sources/platform/actors/publishing/index.mdx

```diff
@@ -5,39 +5,88 @@ sidebar_position: 7.5
+
+To ensure long-term quality and improve your chances of successfully monetizing your Actors, we rocemmend reserving approximately 2 hours per week for maintenance tasks, such as:
+
+- Addressing bug reports & issues
+- Implementing updates & improvements
+- Responding to user inquireies & feedback
 
  <img src={require("./images/apify-store.png").default} title="Apify Store" width="60%" />
 
-If you decide to make your Actor code publicly available on [GitHub](https://github.com), its quality becomes a priority since your Actor could be a new user's first experience with Apify.
+If you decide to make your Actor's code publicly available on [GitHub](https://github.com), code quality becomes even more crucial, as your Actor may be the first experience some users have with Apify.
+
+### Handling breaking changes
+
+While refactoring and updating your Actor's code is encouraged, be cautious of making changes that could break the Actor for existing users. If you plan to introduce breaking change, please contact us at [community@apify.com](mailto:community@apify.com) beforehand, and we'll assist you in communicating the change to your users.
+
+### Documentation and testing
 
-Naturally, your Actor's code should be open to changes and updates, so don't be afraid of refactoring. However, beware of making changes that could potentially break the Actor for its existing users.
+Pay special attention to your Actor's documentation ([README](../../academy/get-most-of-actors/actor-readme)). It should be clear, detailed and readable, using simple language and avoiding technical jargon whenever possible, as your users may not be developers.
 
-If you plan on making a breaking change, please get in touch with us ahead of time at [community@apify.com](mailto:community@apify.com), and we will help you to communicate the change to your users.
+Ensure periodic testing, either manually or by [setting up automatic testing](../development/automated_tests.md) and [monitoring](https://apify.com/apify/monitoring). This can help prevent users from encountering issues with your Actor.
 
-Also, pay special attention to your Actor's documentation ([README](../../academy/get-most-of-actors/actor-readme)). It should be clear, detailed, and readable. Think of the users, who might not be developers, so try to use simple, easy-to-understand language and avoid technical jargon.
+## Inspiration for new Actors
 
-Ensure periodic testing. You can either do it yourself or [set up automatic testing](../development/automated_tests.md) / [monitoring](https://apify.com/apify/monitoring). Try to prevent your users from coming to you with the issues.
+To generate ideas for new Actor, consider the following sources:
```

Comment:
generate => find ?

Reply:
and mention keeping eye on our Discord

Reply:
Good catch, I'll add both the invite link for the Discord community & I think specific link to #hire-freelancers might be good idea, seems like most logical place to look for new ideas.